src/__tests__/Header.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import Header from '../components/header/Header';
+
+it('should render the component Correctly', () => {
+  const header = render(
```

Comment:
## OPTIONAL 
* I strongly suggest that you add more tests to cover as stated in the requirement, 
![image](https://user-images.githubusercontent.com/52497246/217503482-6bf8b5f8-894c-4cd2-91f1-670859de6ef5.png)